src/app/tide/active_peptide_queue.h

```diff
@@ -116,7 +116,7 @@ class ActivePeptideQueue {
 
   // Workspace for computing theoretical peaks for a single peptide.
   // Gets reused for each new peptide.
-  ST_TheoreticalPeakSet theoretical_peak_set_;
+  TheoreticalPeakSetBYSparse theoretical_peak_set_;
```

Comment:
For my own knowledge. What is the difference between ST_TheoreticalPeakSet and TheoreticalPeakSetBYSparse?

Reply:
They are synonyms. See the definition:
typedef TheoreticalPeakSetBYSparse ST_TheoreticalPeakSet; // ST="search time"
from peptide.h:50
I do not know why this was done this way.